src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -97,9 +124,9 @@ function Base.show(io::IO, limiter::SubcellLimiterIDP)
     if !(local_minmax || positivity)
         print(io, "No limiter selected => pure DG method")
     else
-        print(io, "limiter=(")
-        local_minmax && print(io, "min/max limiting, ")
-        positivity && print(io, "positivity")
+        print(io, "Limiter=(")
+        local_minmax && print(io, "Local min/max, ")
+        positivity && print(io, "Positivity, ")
         print(io, "), ")
```

Comment:
Won't this print something like
```
Positivity, ), 
```
?

Reply:
Unfortunately yes. Can you think of a better way to not get an extra comma at the end or beginning with
```
        local_minmax && print(io, "Local min/max, ")
        positivity && print(io, "Positivity, ")
```
Without adding much more lines of code :confused:

Reply:
Thanks a lot :+1:

Reply:
Done in [b776b6f](https://github.com/trixi-framework/Trixi.jl/pull/1738/commits/b776b6f226a05ec2bb6fdcfa5ef10f74c91093f9).